### ISSUE TITLE
Metadata fields AMP Story recent update

### ISSUE THREAD

**Issue by pietrop** [micnews/story-json#9]
From the AMP Story documentation, it looks like Google has updated the Metadata fields for AMP stories with some new ones. 

Was wondering if it was possible (or what it would take) to change the `story-json` to support those?
 
Happy to do a PR, with some guidance, if that helps. 

[More details on the **(4/2018) New metadata requirements** here](https://www.ampproject.org/docs/reference/components/amp-story)

From their docs
>We're adding four new required attributes and two optional attributes to the <amp-story> element:

```html
<!-- `title`, `publisher`, `publisher-logo-src` and `poster-portrait-src` will soon be required. -->
<amp-story standalone title="My Story"
    publisher="The AMP Team"
    publisher-logo-src="https://example.com/logo/1x1.png"
    poster-portrait-src="https://example.com/my-story/poster/3x4.jpg">

<!-- `poster-square-src` and `poster-landscape-src` are optional, but strongly recommended. -->
<amp-story standalone title="My Story"
    publisher="The AMP Team"
    publisher-logo-src="https://example.com/logo/1x1.png"
    poster-portrait-src="https://example.com/my-story/poster/3x4.jpg"
    poster-square-src="https://example.com/my-story/poster/1x1.jpg"
    poster-landscape-src="https://example.com/my-story/poster/4x3.jpg">
```

>These metadata attributes will be used for displaying a preview of this story across the AMP stories ecosystem. For example, these attributes can be used to render an engaging preview link in the bookend of a related story. Providing these attributes will also help ensure your story is future-proof for rich, embedded experiences in AMP stories surfaces to come.

Thanks

**Comment by pietrop** [comment]
ok, cool. thanks @ellell 

Could you advice on whether it would just be a matter of changing the [`render-meta.js` in `story-json-to-amp`](https://github.com/micnews/story-json-to-amp/blob/master/lib/render-meta.js) to accept those fields?

**Comment by ellell** [comment]
@pietrop Not quite, `render-meta.js` is for rendering the schema but for those amp updates we want to add props to the amp-story tag: https://github.com/micnews/story-json-to-amp/blob/master/lib/index.js#L128. But yes it seems like the object passed to render-meta.js is already prepared with at least some of those props so we should reuse the same data.

**Comment by pietrop** [comment]
Hi @ellell,  
Just a heads up we did a pull request https://github.com/micnews/story-json-to-amp/pull/36 in the other repo. 

Let me know what you think
